### ISSUE TITLE
[Privileges 2] "Mixed languages" in CLI

### ISSUE THREAD

**Issue by aduffner** [SAP/macOS-enterprise-privileges#117]
Tested on:
- macOS 15.0.1 (24A348)
- Privileges Beta 2.0.0 (1781)

The localization gets a little mixed up because the German word “Minuten” (minutes) is used after a sentence in the CLI that starts in English. Is the CLI intentionally multilingual?

<img width="850" alt="Bildschirmfoto 2024-10-22 um 10 32 03" src="https://github.com/user-attachments/assets/55e0fad4-1d86-414b-9adf-84f68bcd947e">

In addition, it would be cool if the limit value would dynamically adapt to newer settings and only apply if it is lower than the current limit value (if it comes from a newer profile or a new user setting).

**Comment by mthielemann** [comment]
@aduffner Thanks for reporting this. The language issue has been fixed in build 1791.

As for the automatic adjustment of the remaining time for admin rights, I'm not sure. From a user's perspective, it's a really bad experience when I request admin rights for a certain amount of time and then the time is suddenly reduced for no apparent reason. Technically that wouldn't be a problem, but what would be the practical benefit?

**Comment by aduffner** [comment]
@mthielemann You are right, this would irritate the user and only makes sense in edge cases. An example would be that the user has selected “never” as the expiration date and this should be set to 4 hours, as never is no longer permitted due to a new configuration. Of course it would apply at login/ restart (if set up), but not immediately.

**Comment by mthielemann** [comment]
@aduffner In this edge case you could deploy a profile first that forces standard user privileges (using the `EnforcePrivileges` key) and after this, remove the `EnforcePrivileges`  key and deploy the new expiration interval.
